index.html

```diff
@@ -11,6 +11,68 @@
 </head>
 
 <body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
-</body>
```

Comment:
I think you might have accidentally taken out the container div at the start of your `<body>` tag? You'll need it to prevent all your page content from being spread across the whole width of your browser in desktop view.